xdsl/dialects/affine.py

```diff
@@ -142,6 +146,41 @@ class If(IRDLOperation):
     else_region = region_def()
 
 
+@irdl_op_definition
+class ParallelOp(IRDLOperation):
```

Comment:
nit: link to doc in MLIR?

Reply:
Is that a new convention we have? Otherwise, why specifically here?

Reply:
No, I just felt like it would be useful here specifically, don't see a downside to it also